.github/workflows/build_and_test.yaml

```diff
@@ -52,5 +52,8 @@ jobs:
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
 
       - name: Build and Push EG Dev Image
-        if: github.event_name == 'push'
-        run: make image.push.multiarch TAG=latest IMAGE_NAME=gateway-dev PLATFORMS="linux_amd64 linux_arm64"
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main'
```

Comment:
lets edit this to make it future proof
only push `latest` tag for the `main` branch
```

if: github.event_name == 'push'
# tag is set to the short SHA of the commit
run: make image.push.multiarch PLATFORMS="linux_amd64 linux_arm64"
if: github.event_name == 'push' && github.ref == 'refs/heads/main'
run: make image.push.multiarch TAG=latest PLATFORMS="linux_amd64 linux_arm64"
```